samples/DotNet/Microsoft.ServiceBus.Messaging/.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Delete.

Reply:
Add to .gitignore


---
In reply to: [180609636](https://github.com/Azure/azure-service-bus/pull/185#discussion_r180609636) [](ancestors = 180609636)